rider/backend.gradle

```diff
@@ -20,6 +20,13 @@ task buildReSharperHostPlugin(type: DotNetBuildTask) {
     dependsOn prepareNuGetConfig, generateModels
 
     buildFile = backend.resharperHostPluginSolution
+    doFirst {
+        // "skipDotnet" is easier to type (and remember), but "dotNetUpToDate" is already in use
+        if (skipDotnet.equalsIgnoreCase("true") || dotNetUpToDate.equalsIgnoreCase("true")) {
+            logger.info("Skipping .NET buildReSharperHostPlugin task")
+            throw new StopExecutionException()
+        }
+    }
```

Comment:
How about this? I don't think we need the log, because the default is `false`, so you have to explicitly opt in (via command line parameters - `./gradlew buildPlugin -PskipDotnet=true`).

```
onlyIf { skipDotnet.equalsIgnoreCase("false") && dotNetUpToDate.equalsIgnoreCase("false") }
```

Also, I'd be happy to kill `dotNetUpToDate`. There's no real need in keeping both properties.

Reply:
I want to add a log to describe skip reason.